Tests/Functional/GetTokenTest.php

```diff
@@ -57,7 +57,7 @@ public function testGetTokenFromInvalidCredentials()
         $this->assertArrayHasKey('message', $body, 'The response should have a "message" key containing the failure reason.');
         $this->assertArrayHasKey('code', $body, 'The response should have a "code" key containing the response status code.');
 
-        $this->assertSame('Bad credentials', $body['message']);
+        $this->assertSame('Bad credentials.', $body['message']);
```

Comment:
This is because of [that](https://github.com/symfony/security/blob/master/Core/Authentication/Provider/UserAuthenticationProvider.php#L67)
As `BadCredentialsException` extends `AuthenticationException` and set its message to `Bad credentials.` then the old assert is wrong now